server/src/test/java/io/spine/examples/shareaware/server/e2e/given/E2EUser.java

```diff
@@ -113,17 +114,18 @@ public WalletId walletId() {
      * Describes the user's action to look at available shares on the market.
      */
     public List<Share> looksAtAvailableShares() {
-        var shares = availableMarketShares
-                .state()
-                .getShareList();
-        return shares;
+        var marketShares = availableMarketShares.state();
+        Preconditions.checkNotNull(marketShares);
```

Comment:
This should be imported statically. 

And also, that should probably be `requireNonNull`, which is a part of JDK.

Same below.